src/components/cards/EvaluationInfo.vue

```diff
@@ -139,39 +142,68 @@ export default class EvaluationInfo extends EvaluationInfoProps {
   }
 
   watch() {
-    console.log("Prop changed: " + this.filter);
+    console.log("Prop changed: " + this.file_filter);
   }
 
-  get filename() {
-    let store = getModule(InspecDataModule, this.$store);
-    let file = store.allFiles.find(f => f.unique_id === this.filter);
-    if (file) {
-      let eva = file as EvaluationFile;
-      console.log("filename 1: " + eva.filename);
-      if (eva.database_id === null) {
-        console.log("no database id");
-      } else {
-        this.database_id = eva.database_id || null;
-        this.load_tags(this.database_id);
-      }
-      return eva.filename;
-    } else {
-      return null;
+  get filename(): string {
+    return this.file.filename;
+  }
+
+  get inspec_version(): string {
+    return this.file.evaluation.data.version;
+  }
+
+  get platform(): string {
+    return (
+      this.file.evaluation.data.platform.name +
+      this.file.evaluation.data.platform.release
+    );
+  }
+
+  get get_duration(): string {
+    return this.file.evaluation.data.statistics.duration + "";
+  }
+
+  get file(): EvaluationFile {
+    console.log("file filter: " + this.file_filter);
+    let filter_module = getModule(FilteredDataModule, this.$store);
+    return filter_module.evaluations([this.file_filter])[0].from_file;
+  }
+
+  get width(): number {
+    let mod = getModule(ServerModule, this.$store);
+    if (mod.serverMode == undefined) {
+      mod.server_mode();
+    }
+    if (mod.serverMode) {
+      return 500;
```

Comment:
Some sort of comment explaining why these numbers were chosen would be nice, without reading the function name at first glance I thought this was returning an HTTP error code. At least a comment explaining the reason for each of these numbers will make it easier for future developers.